Source/WebCore/platform/SharedBufferChunkReader.cpp

```diff
@@ -62,7 +64,7 @@ void SharedBufferChunkReader::setSeparator(const Vector<char>& separator)
 void SharedBufferChunkReader::setSeparator(const char* separator)
 {
     m_separator.clear();
-    m_separator.append(span(separator));
+    m_separator.append(WTF::span(separator));
```

Comment:
(Note: without the explicit use of WTF namespace here, Clang tries to use the new `WebCore::span` instead and complains that it cannot cast to the Skia type.)